application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -232,8 +263,23 @@ private void handleActionWithMessage(@NotNull TagStatus requiredTagStatus,
         }
 
         event.getMessageChannel().retrieveMessageById(messageId).queue(message -> {
+            String previousContent = null;
+            if (subcommand == Subcommand.EDIT_WITH_MESSAGE) {
+                try {
+                    previousContent = tagSystem.getTag(tagId).orElseThrow();
+                } catch (NoSuchElementException exception) {
+                    logger.debug(
+                            "tried to retrieve previous content of tag '%s', but the content doesn't exist.");
+                }
+            }
```

Comment:
see other comment (maybe this should be moved into a helper method instead to get rid of the duplication)

Reply:
done, 4663b308d9b99af4162ab4f3bc9c1fac9655b68d